### ISSUE TITLE
Not working at www.curseforge.com

### ISSUE THREAD

**Issue by pkejval** [RyuzakiH/CloudflareSolverRe#7]
URL: https://www.curseforge.com/wow/addons/deadly-boss-mods

This code fails with exception **CloudflareSolverRe.Exceptions.CloudflareClearanceException**: Clearance failed after 3 attempt(s).

Not working even if I increase number of MaxTries nor incereasing ClearanceDelay.

But when I try URL from this project tests, everything is working. Maybe www.curseforge.com uses new protection? 

Pythons library https://pypi.org/project/cfscrape/ can overpass even this Cloudflare protection so it's doable but I don't know how. :)

```
            var target = new Uri("https://www.curseforge.com/wow/addons/deadly-boss-mods");

            var handler = new ClearanceHandler
            {
                MaxTries = 3,
                ClearanceDelay = 3000
            };

            try
            {
                var client = new HttpClient(handler);
                var content = await client.GetStringAsync(target);
            }
            catch (Exception ex) { Console.WriteLine(ex.ToString()); }
```

**Comment by N0ise1337** [comment]
Yes, CFScrape works fine with that, i did a test using a small script and worked fine
However the websites doesn't contain uam or recaptcha, i guess thats made for uam / captcha only

**Comment by RyuzakiH** [comment]
Sorry for late reply, but this site seems to use the recaptcha challenge every time for me.
I think cfscrape uses a way to decrease the recaptcha challenge possibility.
This library cannot make this now, I'll try finding a way.

**Comment by Summpot** [comment]
This python library doesn't handle captcha at all, it prevents the appearance of captcha by adding ciphers to a urllib3_ssl_context. Is there something similar in C # that can implement this function?
